include/tvm/relax/analysis.h

```diff
@@ -260,6 +260,14 @@ TVM_DLL bool IsBaseOf(const StructInfo& base, const StructInfo& derived,
 TVM_DLL StructInfo StructInfoLCA(const StructInfo& lhs, const StructInfo& rhs,
                                  arith::Analyzer* ana = nullptr);
 
+/*!
+ * \brief Get the symbolic TIR variables that appear in the input struct info.
+ * The returned list is deduplicated - each TIR variable will appear at most once.
+ * \param sinfo The struct info object to be analyzed.
+ * \return The list of TIR variables that appear in the input struct info.
+ */
+TVM_DLL Array<tir::Var> SymVarInStructInfo(const StructInfo& sinfo);
```

Comment:
TIRVarInStructInfo , so we don't need to learn a new terminology

Reply:
Updated.